packages/engine/src/scene/functions/loaders/AudioFunctions.test.ts

```diff
@@ -485,6 +492,13 @@ describe('AudioFunctions', () => {
 
       assert(audioEl.isPlaying !== prevState)
     })
+
+    it('should do nothing if no audio component is there', async () => {
+      const entity = createEntity()
+      addComponent(entity, Object3DComponent, { value: new Object3D() })
+      audioFunctions.toggleAudio(entity)
+      assert(true)
```

Comment:
missing assertion

Reply:
This test case is checking whether there is `audioEl` present in `object3d.userData`. If there is no `audioEl` then function should not throw any error. So a successful return from the `toggleAudio` function is in itself a assertion that this test is passed.